File: 100-Days-Series/Day 94/ReplaceNegativeNumberWithZero.java
public class ReplaceNegativeNumberWithZero {
    // Method to replace negative numbers with zero.
    public static void getReplaceNegativeNumberWithZero(int arr[]){
        for(int i=0;i<arr.length;i++){
            if(arr[i]<0){
                arr[i]=0;
            }
        }
    }
    // Method to print an array
    public static void getPrintArray(int arr[]){
        for(int i=0;i<arr.length;i++){
            System.out.print(arr[i]+" ");
        }
        System.out.println();
    }
    // Main Method 
    public static void main(String [] args){
        int arr[]={-1, 2, -3, 4, -5};
        System.out.println("Original Array");
        getPrintArray(arr);
        getReplaceNegativeNumberWithZero(arr);
        System.out.println("Updated Array");
        getPrintArray(arr);
    }

}

File: 100-Days-Series/Day 94/ReplaceNegativeNumberWithZero.cpp
#include<iostream>
#include<vector>
using namespace std;
// Function to Replace Negative number with zero.
void getReplaceNegativeNumbetWithZero(vector<int>&arr){
    int size=arr.size();
    for(int i=0;i<size;i++){
        if(arr.at(i)<0){
            arr.at(i)=0;
        }
    }
}
// Function to print an array.
void getPrintArray(vector<int>&arr){
    for(int i=0;i<arr.size();i++){
        cout<<arr.at(i)<<" ";
    }
    printf("\n");
}
// Let's Start Here
int main(){
    vector<int>arr={-1, 2, -3, 4, -5};
    cout<<"\nOriginal Array"<<endl;
    getPrintArray(arr);
    getReplaceNegativeNumbetWithZero(arr);
    cout<<"\nUpdated Array"<<endl;
    getPrintArray(arr);

}